index.html

```diff
@@ -11,64 +11,66 @@
+            <header>
+                <a href="#" id="logo">My Logo</a>
+                <nav>
+                    <i class="fa fa-bars fa-lg"></i>
+                    <ul class="nav-links">
+                        <li>
+                            <a href="#card-section">Portfolio</a>
+                        </li>
+                        <li>
+                            <a href="#about">About</a>
+                        </li>
+                        <li>
+                            <a href="#form-section">Contact</a>
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+            <section class="main-container">
+                <div class="main-content-intro">
+                    <div>
+                        <h1>I'm Moyasi. <br />
+                            Glad to see you!
+                        </h1>
+                    </div>
+                    <div>
+                        <p>
+                            I'm a software developer! I can help you build a product , feature or website Look through some
+                            of my work and experience!
+                            If you like what you see and have a project you need coded, don't hestiate to contact me.
+                        </p>
+                    </div>
+                    <div id="social__container">
+                        <a href="#" id="social-heading">Let's Connect</a>
+                            <div class="social__icons">
+                                <ul>
+                                    <li>
+                                        <a href="https://twitter.com/moyasi_ginko"><img src="asset/twitter.svg"
+                                                alt="twitter icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="https://www.linkedin.com/in/mahmudur-rahman-a8a151257/"><img
+                                                src="asset/Linkedin icon.svg" alt="Linkedin icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="#"><img src="asset/M.svg" alt="M icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="https://github.com/MoyasiGinko"><img src="asset/github.svg"
+                                                alt="github icon"></a>
+                                    </li>
+                                    <li>
+                                        <a href="#"><img src="asset/thumbs.svg" alt="thumbs icon"></a>
+                                    </li>
+                                </ul>
+                            </div>
+                    </div>
                 </div>
-            </div>
-        </section>
+            </section>
+        </div>
         <section id="card-section">
```

Comment:
- [ ] In the cards container, half of it contains the description, and half of it contains the picture, and in each card this discipline switch places (in first card, the picture on the left and the description on the right, in the next card the description on the left and the picture on the right and so on and so forth), but in your design, card no 3 and 4 contradict this rule so kindly consider fixing it.
your design
![re](https://user-images.githubusercontent.com/96683843/230521302-4ddc7dd0-c80a-4a04-b1f3-6cf61a99d90e.jpg)

figma design
![p](https://user-images.githubusercontent.com/96683843/230521359-b59784f0-ca58-40f8-b2a5-50d740f00113.jpg)